contribution/lang/fr.json

```diff
@@ -145,35 +145,35 @@
  	 	 	},
  	 	 	"toStation": {
  	 	 	 	"name": {
- 	 	 	 	 	"trans": "Aller vers la station d'Hypertrain",
+ 	 	 	 	 	"trans": "Aller vers la Station Centrale d'Hypertrain",
  	 	 	 	 	"eng": "Go to Hyper Train Central Station"
  	 	 	 	},
  	 	 	 	"tut": {
- 	 	 	 	 	"trans": "Débutez votre aventure vers les récompenses épiques ici",
+ 	 	 	 	 	"trans": "Débutez votre aventure pour des récompenses épiques ici",
```

Comment:
last one was better

Reply:
récompenses is not a destination, thus it is not correct to use "vers". "récompenses" are goods you get, they are objects. using "pour" is better